_docs-sources/intro/overview/intro-to-gruntwork.md

```diff
@@ -0,0 +1,17 @@
+# Introduction to Gruntwork
+
+### What is Gruntwork
+
+Gruntwork is a "DevOps accelerator" designed to make it possible to achieve a world-class DevOps setup based completely on infrastructure-as-code in just a few days.
+
+Gruntwork works best for teams building new infrastructure ("greenfield"), whether from scratch or as part of a migration, however it can also be useful for teams with existing infrastructure ("brownfield").
+
+The core elements of a world-class DevOps setup are described in the [Gruntwork Production Framework](gruntwork-production-framework). The [Gruntwork product suite](products) is designed to help software teams implement the Gruntwork Production Framework.
+
+### Supported public clouds
+
+Gruntwork accelerates teams on Amazon Web Services (AWS). Support for additional public clouds such as GCP and Azure may be added in the future.
+
+### Gruntwork and Terraform
+
+All Gruntwork products are built on and fully compatible with [open source Terraform](https://gruntwork.io). The one exception to this is the Gruntwork Reference Architecture, which uses Terragrunt to implement an end-to-end architecture.
```

Comment:
We should link to **Terragrunt** here, especially since that's a new tool for many readers.